### DISCUSSION TITLE
Problems updating firmware

### DISCUSSION THREAD

**Discussion by s0129** [john30/ebusd#1019] [General]
I have been attempting to update the firmware using the web browser via https://adapter.ebusd.eu/v5/firmware.en which has previously worked. However recently I am not getting the install popup anymore. Using developer tools I can see there is a 404 when fetching https://adapter.ebusd.eu/v5/flash/install-dialog-2aee1312.js
Is there a problem with the website?

**Comment by john30** [comment]
probably a reload in your browser is needed

**Reply by s0129** [reply]
Have tried reload of browser, different browsers and different PCs. All the same issue.

<img width="425" alt="image" src="https://github.com/john30/ebusd/assets/1311577/8d9b25d0-e915-4643-bdd4-6df1769a50a1">

**Comment by john30** [comment]
you need to disable the cache or something in order to make it load the up-to-date install-button.js file

**Reply by s0129** [reply]
That did the trick! Thanks.
